src/components/AComp/_acomp.scss

```diff
@@ -9,14 +11,14 @@
   z-index: 1;
 }
 
-.acomp__title {
-  color: white;
-}
+  .acomp__title {
```

Comment:
I'm assuming you've added the indents here to indicate descendant selectors of the main component class. I think it's perhaps unnecessary as the BEM format will communicate that, plus we're aiming to nest as little as is necessary

Reply:
> assuming you've added the indents here to indicate descendant selectors

Yep. 

> we're aiming to nest the selectors as little as possible

With both our browser support needs (IE11...) and the messy realities of writing CSS do you think it's likely that we will need to write selectors that nest? If that is the case, I think the indents give us good scope to handle this. If we look at the following example...

```css
.c-component {}

.c-component__title {}

.c-component__body {}

.c-component__body-title {}
```

With BEM, it looks like `.c-component__body-title` is an element of the `.c-component` block, but in reality it's dependent on the `.c-component__body` block. Whilst this does break an explicit BEM rule, I've found that sometimes it is necessary to do so to work with unwieldy markup or even just to get the desired design. Indenting would help us see immediately the BEM model, even whilst breaking (or _embellishing on_ 😇) a BEM rule.

(I would personally want to write the above in the following way):

```css
.c-component {}

  .c-component__title {}

  .c-component-body {}

    .c-component-body__title {}
```

---

Sidnote: Even ignoring the above arguments - for me the indents help as they make it immediately clear the hierarchy of selectors, embellishing on the BEM conventions!

Interested to hear your thoughts?

Reply:
Resolve to using indents providing they play nicely with stylelint https://github.com/wellcometrust/corporate/issues/5672